### ISSUE TITLE
[BRUKERHISTORIE] Dokumentasjon av søk

### ISSUE THREAD

**Issue by livarb** [datadeling/behov-issue-tracker#137]
Som: bruker av datafabrikken sitt søk
Ønsker jeg å: vite hvordan søket fungerer
Slik at: jeg kan bruke søket rett og benytte meg av avanserte søkefunksjoner

------------------

Relatert til epos: #84 

Lenke til design:
Lenke til løsningsarkitektur:

Forretningsverdi (Kritisk/Viktig/Mindre viktig/Kjekt å ha):

**Minimumsløysing**
Skriv nokre setningar om korleis søket fungerer her:
https://informasjonsforvaltning.github.io/felles-datakatalog/s%C3%B8k_og_filter/

Moment å ta med:
- Er det AND eller OR når fleire søkeord er tasta inn?
- Kva søketeknologi ligg under (f.eks. ElasticSearch sett til norsk språk, med stemming).
- Kva felt blir søkt i
- Kan ein søke på fraser (dvs. heile setningar omslutta av hermeteikn)
- Kjente begrensingar
- Gjerne direktelenke(r) til kjeldekoden der søkeoppsettet er

**Bakgrunn**
I dag er det ikkje dokumentert korleis søket fungerer.
For eksempel, kan eg bruke AND og OR i søkefeltet for å styre søket, kan eg omslutte ei setning med hermeteikn for å søke på fraser? Korleis fungerer vektning i søkeresultata?

**Ingen dokumentasjon i FDK**
Det finst ein plasshaldar i FDK sin dokumentasjon, men dette er ikkje fylt ut.
![image](https://user-images.githubusercontent.com/3617256/147207280-5a31fc6d-cea8-4d71-b68b-618f34dea2b9.png)
https://informasjonsforvaltning.github.io/felles-datakatalog/s%C3%B8k_og_filter/

**Eksempel på dokumentasjon hos andre**
Eksempel på dokumentasjon av søk hos Finn.no:
https://hjelpesenter.finn.no/hc/no/articles/303324-Tips-om-friteksts%C3%B8k-og-ord-i-annonsen-
![image](https://user-images.githubusercontent.com/3617256/148502812-ee93e119-20ab-4b16-9f32-886d4cdc7960.png)

**Comment by livarb** [comment]
@shenluna : kva er betydninga av den nye tittelen på saka?

**Comment by livarb** [comment]
@shenluna : Treng meir informasjon i dette issuet. Det var snakk om å skrive Gherkin-testar?
Lenk gjerne til relevante eksempel på andre slike testar.

**Comment by shenluna** [comment]
@livarb Ja, det stemmer, jeg tror dessverre ikke vi har noen eksempler på slike tester i repoene våre per dags dato.

**Comment by livarb** [comment]
@stigbd : Forstod det slik at du er hjernen bak planen om å skrive testar.
Kan du forklare meir korleis dette vil fungere? Eksempel på testar, korleis vil det sjå ut for sluttbrukar til slutt - som dokumentasjon på korleis søket fungerer.

**Comment by stigbd** [comment]
Det eg har i tankane når det gjelder akseptanse-test av søk, er gherkin. Eit enkelt døme her:
https://automationpanda.com/2017/01/27/bdd-101-gherkin-by-example/

Sluttbrukardokumentasjon er noko anna, men akseptanse-testen kan vere input til sluttbrukardokumentasjonen.

**Comment by stigbd** [comment]
Ser at tittel på issuet og innholdet i brukerhistorien adresserer ulike ting. Det bør antagelegvis lagast to issues.

**Comment by livarb** [comment]
@stigbd : Eg har oppretta nytt issue angåande akseptansetest. Fyll gjerne inn fleire detaljar.

**Comment by livarb** [comment]
Foreslår minimumsløysing for dokumentasjon av søk:
Skriv nokre setningar om korleis søket fungerer her:
https://informasjonsforvaltning.github.io/felles-datakatalog/s%C3%B8k_og_filter/

Moment å ta med:
- Er det AND eller OR når fleire søkeord er tasta inn?
- Kva søketeknologi ligg under (f.eks. ElasticSearch sett til norsk språk, med stemming).
- Kva felt blir søkt i
- Kan ein søke på fraser (dvs. heile setningar omslutta av hermeteikn)
- Kjente begrensingar
- Gjerne direktelenke(r) til kjeldekoden der søkeoppsettet er

**Comment by livarb** [comment]
Oppdaterte beskrivelsen av saka med forslag til minimumsløysing, så kan vi drøfte dette ved første anledning. Bør vere ein relativt overkommeleg jobb.

**Comment by livarb** [comment]
Flytta denne til «Avklare» i påvente av prioritering framover.
